src/Miunie.Discord/DSharpPlusDiscord.cs

```diff
@@ -126,6 +126,15 @@ public async Task SendMessageAsync(MiunieChannel mc, DirectoryListing dl)
             await channel.SendMessageAsync(result);
         }
 
+        public async Task SendMessageAsync(MiunieChannel mc, TranslateRequest[] transRequests)
```

Comment:
This is way too specific for an Adapter.
The concept of a TranslateRequest isn't based in any of the underlying structure.

When something is a phrase the act of getting it from a phrase key is not translation, moreover, it will cause confusion once actual translation is implemented.

This should no have bubbled up this far. Should have been handled by a provider and packed into a single object for processing.

Reply:
Yup I was afraid this wouldn't fit in the current architecture, but I'm not sure if I understand what do you suggest, does that mean to inject the `ILanguageProvider` in the `UserReputationProvider` to resolve the _act of getting a phrase from a phrase key_ in the reputation provider?